tests/tests.py

```diff
@@ -399,6 +400,49 @@ def test_index_can_hide_all_emails(self, mock_settings):
         self.assertNotIn('alpha@alpha.alpha', response.content)
         self.assertNotIn('bravo@bravo.bravo', response.content)
 
+    def test_check_requires_permission(self):
+        self.client.login(username='superuser', password='superuser')
+        response = self.client.get(reverse('invite:check'))
+        self.assertTrue(response.status_code == 200)
+        self.client.logout()
+        self.client.login(username='normal', password='normal')
+        response = self.client.get(reverse('invite:check'))
+        self.assertTrue(response.status_code == 403)
+
+    def test_check_returns_true_when_taken(self):
+        self.client.login(username='superuser', password='superuser')
+        taken = (
+            {'username': self.normal_user.username},
+            {'username': self.supertwo.username},
+            {'email': self.normal_user.email},
+            {'email': self.supertwo.email},
+        )
+        for each in taken:
+            response = self.client.get(reverse('invite:check'), each)
+            self.assertTrue(json.loads(response.content)['taken'] is True)
+
+    def test_check_returns_false_when_not_taken(self):
+        self.client.login(username='superuser', password='superuser')
+        not_taken = (
+            {'username': 'doesnotexist'},
+            {'username': ''},
+            {'email': 'nothing@nothing.nothing'},
+            {'email': 'test@test.test'},
+        )
+        for each in not_taken:
+            response = self.client.get(reverse('invite:check'), each)
+            self.assertTrue(json.loads(response.content)['taken'] is False)
```

Comment:
You could do `self.assertFalse(json.loads(response.content)['taken'])` instead.

Reply:
yeah I'll change this and the next instance